py/kubeflow/kfctl/testing/pytests/kf_is_ready_test.py

```diff
@@ -259,6 +262,17 @@ def test_kf_is_ready(record_xml_attribute, namespace, use_basic_auth,
             util.run(["kubectl", "-n", ss_namespace, "describe", "statefulsets", name])
             raise
 
+    all_daemon_sets = [(namespace, name) for name in daemon_set_names]
```

Comment:
Remind me does kubeflow manifest install any daemonset?

Reply:
Yes for aws we have nvidia plugin, https://github.com/kubeflow/manifests/blob/master/aws/nvidia-device-plugin/base/daemonset.yaml